2.4/root/usr/bin/run-mongod

```diff
@@ -63,12 +49,15 @@ if [ $# -gt 0 ] && [ "$1" == "initiate" ]; then
   exec ${CONTAINER_SCRIPTS_PATH}/initiate_replica.sh
 fi
 
-# Generate config file for MongoDB
-envsubst < ${CONTAINER_SCRIPTS_PATH}/mongodb.conf.template > $MONGODB_CONFIG_PATH
+# If user provides own config file use it and do not generate new one
+if [ ! -s $MONGODB_CONFIG_PATH ]; then
+  # Generate config file for MongoDB
+  envsubst < ${CONTAINER_SCRIPTS_PATH}/mongodb.conf.template > $MONGODB_CONFIG_PATH
+fi
 
 # Need to cache the container address for the cleanup
 cache_container_addr
-mongo_common_args="-f $MONGODB_CONFIG_PATH --oplogSize 64"
+mongo_common_args="-f $MONGODB_CONFIG_PATH --dbpath $MONGODB_DATADIR"
```

Comment:
To avoid getting values from config file - https://github.com/openshift/mongodb/pull/147#discussion_r61854214

`dbpath` is one example and it is really necessary for MongoDB scripts. Also for better performance database should be stored in the docker volume. And user mounting config files from "classical" deployments don't have to be aware of it.

So explicit `--dbpath` overrides value from config file.